homeworks/egetoimre/3_conditionals_loops/flat_finder.py

```diff
@@ -0,0 +1,15 @@
+town = input("Melyik városban található az albérlet?")
+price = int(input("Mennyibe kerül az albérlet havonta?"))
+
+if (town == "New York" or town == "San Fransisco") and price < 4000:
```

Comment:
Nagyon szuper az egész megoldásod ehhez a feladathoz, ezt a kommentet se vedd hibának, csak egy apró egyszerűsítési lehetőség:

`if town in ['New york', 'San Francisco'] and price < 4000:`

az `in` operátorral nem kell az or és a két town összehasonlítás.